WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Package and Rate Selection/WooShippingAddPackageView.swift

```diff
@@ -0,0 +1,254 @@
+import SwiftUI
+
+struct WooShippingAddPackageView: View {
+    enum PackageProviderType: CaseIterable {
+        case custom, carrier, saved
+        var name: String {
+            switch self {
+            case .custom:
+                return Localization.custom
+            case .carrier:
+                return Localization.carrier
+            case .saved:
+                return Localization.saved
+            }
+        }
+    }
+
+    @Environment(\.presentationMode) var presentationMode
+
+    @State var selectedPackageType = PackageProviderType.custom
+    var addPackageButtonDisabled: Bool {
+        for (_, value) in fieldValues {
+            if value.isEmpty {
+                return true
+            }
+        }
+        return fieldValues.count != WooShippingAddPackageDimensionView.DimensionType.allCases.count
+    }
+
+    var body: some View {
+        NavigationView {
+            VStack {
+                Picker("", selection: $selectedPackageType) {
+                    ForEach(PackageProviderType.allCases, id: \.self) {
+                        Text($0.name)
+                    }
+                }
+                .pickerStyle(.segmented)
+                .padding()
+                Spacer()
```

Comment:
Is this `Spacer()` needed? It looks like there's more space between the picker and the rest of the view than in the design:

Design|Actual
-|-
![image](https://github.com/user-attachments/assets/15f2ac08-7b4b-4053-abbc-a969a83bb81b)|![Simulator Screenshot - iPhone 16 Pro - 2024-10-01 at 16 10 03](https://github.com/user-attachments/assets/858cd36b-e4bc-4d17-af09-cfda381ea43c)